src/systemctl/systemctl-logind.c

```diff
@@ -14,6 +14,8 @@
 #include "terminal-util.h"
 #include "user-util.h"
 
+#define SD_LOGIND_ROOT_CHECK_INHIBITORS  (UINT64_C(1) << 0)
```

Comment:
Not sure the proper way to share headers between src/login and src/systemctl. Any pointer?

Reply:
see the enum UnitFileFlags in src/shared/install.h for an example of a flag used in a DBUS interface and shared. A good place to store this would probably be the src/shared/reboot-util.h header.